lib/utils/database_helpers.rb

```diff
@@ -48,10 +48,22 @@ def successful?
       @cmd.exit_status == 0 && @error.nil?
     end
 
+    def rows
+      @results
+    end
+
     def row(id)
       SQLRow.new(self, @results[id])
     end
 
+    def column(column)
+      result = []
+      @results.each do |row|
+        result << row[column] if row.has_key?(column)
```

Comment:
Why the conditional here?  RDBMS query results are always regular; column presence can't vary by row.

Reply:
Yes, it is just an empty glass for the case I don't wanna drink :)
To be serious - if provided column name  does not exists it will cause a ruby error, with if it will result in just empty array.

Reply:
I think you might want to fail the resource if the column doesn't exist.  That would almost certainly indicate an error in the query, which the user should know about and fix.  An empty array could be interpreted as a normal empty result.